### ISSUE TITLE
NTR: Manganese high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8872]
New term request Manganese high in urine (elthat_160116164812):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000480
- label: Elevated urine manganese level
- parents: [Abnormal urinary mineral level (HP:0025640)](https://hpo.jax.org/app/browse/term/HP:0025640)
- pmid: PMID:1577529
- def: The amount of manganese in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: Manganese is a trace mineral required for metabolism, growth and tissue formation, and reproduction. It is mainly obtained through food and water, as well as through occupational exposure.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8872
- synonyms:
